app/controllers/followups_controller.rb

```diff
@@ -76,7 +76,9 @@ def check_request_matches_incoming_message
 
   def check_responses_allowed
     if @info_request.allow_new_responses_from == "nobody"
-      flash[:error] = _('Your follow up has not been sent because this request has been stopped to prevent spam. Please <a href="{{url}}">contact us</a> if you really want to send a follow up message.', :url => help_contact_path.html_safe)
+      flash[:error] = { :partial => "followup_not_sent.html.erb",
```

Comment:
I think this should be `flash.now[:error]` ? There's no redirect, so the flash will persist until the next request.